tests/fixture_test.go

```diff
@@ -210,15 +208,50 @@ func (fixture *Fixture) EventBody() []byte {
 }
 
 func (fixture *Fixture) WebhookHmacSignature() string {
-	return readFixtureData().WebhookHmacSignatureString
+	return readFixtureData().Webhooks["hmac_signature"].(string)
 }
 
 func (fixture *Fixture) WebhookSecret() string {
-	return readFixtureData().WebhookSecretString
+	return readFixtureData().Webhooks["secret"].(string)
 }
 
 func (fixture *Fixture) WebhookUrl() string {
-	return readFixtureData().WebhookURL
+	return readFixtureData().Webhooks["url"].(string)
+}
+
+func (fixture *Fixture) WebhookCustomHeaders() ([]easypost.WebhookCustomHeader, error) {
```

Comment:
thought: I don't believe we need all this logic and especially not the error handling. These fixtures only intended purpose are to load data from the examples submodule files. We'd expect them to be properly formatted when accessed and therefore shouldn't need all the logic here.